languages/swift/exercises/concept/functions/README.md

```diff
@@ -0,0 +1,63 @@
+In this exercise you're going to write some more code to help you cook your brilliant lasagna from your favorite cooking book.
```

Comment:
I really like how you've expanded on the `basics` exercise's story in the tasks!